Cython/Compiler/Nodes.py

```diff
@@ -7794,10 +7794,12 @@ class GILStatNode(NogilTryFinallyStatNode):
     #
     #   state   string   'gil' or 'nogil'
 
+    child_attrs = ["body", "condition", "finally_clause", "finally_except_clause"]
```

Comment:
I'd spell this
`child_attrs = ["condition"] + NogilTryFinallyStatNode.child_attrs`
Just to avoid accidental inconsistencies in the future.

Reply:
Fixed.